system-addon/content-src/components/ContextMenu/_ContextMenu.scss

```diff
@@ -50,6 +50,11 @@
             color: $white;
           }
         }
+
+        &.disabled {
+          opacity: 0.4;
+          cursor: default;
```

Comment:
I believe if we make it `pointer-events: none` there would be no click event to specially handle anyway?

Reply:
I had it like that but then the click would still bubble up to window and the context menu would hide. And I don't think that's the expected behavior.

Reply:
Ah okay. Although.. there is a separate bug #3251 about clicking the separator causing the menu to dismiss..... I suppose a larger fix would be to eat events on the menu that aren't for actionable options.

Reply:
Oh hmm. Maybe the context menu container can stop propagation on all clicks. And clicks to valid items can call `hideContext()` directly. Will try

Reply:
da15d8a  seems to work 💃